qiskit_ibm_provider/ibm_backend.py

```diff
@@ -257,19 +257,25 @@ def __getattr__(self, name: str) -> Any:
                 )
             )
 
-    def _get_properties(self) -> None:
+    def _get_properties(self, datetime: Optional[python_datetime] = None) -> None:
         """Gets backend properties and decodes it"""
         if not self._properties:
-            api_properties = self.provider._runtime_client.backend_properties(self.name)
+            if datetime:
```

Comment:
I think the issue is that when `target_history` is called, there can be properties already cached so the properties used will not be updated even if a datetime is passed in. The same goes for getting the defaults

Reply:
Thanks. I actually see (using prints) that the program reaches `Backend.pulse_defaults` and line 94: `return self.session.get(url, params=params).json()` in `rest/backend.py`. So I don't think that is the problem.

Reply:
I pulled this branch and the following returns the correct target:

```python
datetime = datetime.datetime(2021, 8, 30)
backend = provider.get_backend("ibmq_montreal")
backend.target_history(datetime=datetime)
```

Reply:
That's good news! at some point I did see the correct target, but after I did the defaults, I wasn't so sure any more. Can you try the defaults as well? And how are you sure you are getting correct results? My assumption was that values should be different than the current ones.

Reply:
Yeah for `target`, you can compare the results to see if they're different. Also Matthew left a comment [here](https://runtime-us-east.quantum-computing.ibm.com/openapi#/Backends/get_backend_defaults) for a way to verify the values are correct. 

Ah ok I just noticed - for `defaults` it's not working because the api currently doesn't not support a date parameter 
https://runtime-us-east.quantum-computing.ibm.com/openapi#/Backends/get_backend_defaults

If we want to support `defaults` it would require an update on the server side

Reply:
I see. 

the link you sent for the server API is very helpful.
So shall I cancel the changes I made for `defaults`?
And should I copy the changes for `target` to `qiskit-ibm-runtime` repository?

Reply:
Yeah we can hold off on the updates to `defaults` for now and yes, it would be good to have these changes in qiskit-ibm-runtime too

Reply:
I changed to support `target` only. I tested in my own environment, but did not add a test, because if I understand correctly, this is only relevant for real devices.